libbeat/outputs/kafka/client.go

```diff
@@ -434,3 +452,13 @@ func (c *client) Test(d testing.Driver) {
 	}
 
 }
+
+func isAuthError(err error) bool {
+	for _, e := range authErros {
+		if errors.Is(err, e) {
+			return true
+		}
+	}
+
+	return false
+}
```

Comment:
I thought there was an errros function equivalent to this, but it is still a proposal: https://github.com/golang/go/issues/70961.

Would be neat to say `errors.In(err, authErrors)` instead. Maybe at some point in the future.

Reply:
I'd love that too!